lib/FlagDependencyUsagePlugin.js

```diff
@@ -86,9 +86,9 @@ class FlagDependencyUsagePlugin {
 					}
 
 					const queue = [];
-					for (const chunk of compilation.chunks) {
-						if (chunk.entryModule) {
-							processModule(chunk.entryModule, true);
+					for (const preparedEntrypoint of compilation._preparedEntrypoints) {
```

Comment:
It is possible to use `compilation.entrypoints` instead?

Reply:
Sadly not, as this plugin runs before these property is set. But we should make `_preparedEntrypoints` public in the next major version

Reply:
Thanks for clarification. Since `_preparedEntrypoints` is a private API, isn't it safe to change it now?

Reply:
Technically yes, but I better don't risk it...